packages/react-components/react-progress/stories/ProgressBar/ProgressBarMax.stories.tsx

```diff
@@ -1,8 +1,26 @@
 import * as React from 'react';
 import { ProgressBar } from '@fluentui/react-progress';
+import { Field } from '@fluentui/react-field';
```

Comment:
_nit:_ Alphabetically order import packages

```suggestion
import { Field } from '@fluentui/react-field';
import { ProgressBar, ProgressBarProps } from '@fluentui/react-progress';
```